tests/sav/hello_ios.res

```diff
@@ -0,0 +1 @@
+hello_ios.bin is a directory
```

Comment:
If there is a way to have some kind of reproductible metadata about an app we can use that instead of `is a directory`.
For instance, what about the content of the directory ( `find $dir | sort` ) or what is the answer of `file` on it, or the output of some low-level macosx tool.